src/main/resources/validationrules/iac/iac-judicial-org-role-mapping.drl

```diff
@@ -224,8 +222,12 @@ then
    attribute.put("jurisdiction", JacksonUtils.convertObjectIntoJsonNode("IA"));
    attribute.put("region", JacksonUtils.convertObjectIntoJsonNode($joh.getRegionId()));
    attribute.put("baseLocation", JacksonUtils.convertObjectIntoJsonNode($joh.getBaseLocationId()));
-   attribute.put("primaryLocation", JacksonUtils.convertObjectIntoJsonNode($joh.getPrimaryLocation()));
-   attribute.put("contractType", JacksonUtils.convertObjectIntoJsonNode("Salaried"));
+   attribute.put("substantive", JacksonUtils.convertObjectIntoJsonNode("Y"));
+   if($joh.getOffice().equals("IAC Tribunal Judge (Fee-Paid)")) {
```

Comment:
Please remove  if condition to check contract type as judge role is  only applicable for Tribunal Judge(Salaried)
Just keep this attribute.put("contractType", JacksonUtils.convertObjectIntoJsonNode("Salaried"));